### PR TITLE
Ret 793

### PR DESCRIPTION
**Pull request by marcjudge**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

[### JIRA link ###]

https://tools.hmcts.net/jira/browse/RET-793

### Change description ###
ET1 Form Development - Post create account landing page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/views/new-account-landing.njk [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </div>
+    </div>
+  </div>
+{% endblock %} 
+
+{% block footer %}
+  <div class="govuk-width-container">
+    <div class="govuk-button-group">
+      {{ govukButton({
+        text: "Continue",
```
Comment (Angela-LW) [inline src/main/views/new-account-landing.njk:29]: should use the 'continue' string from the common.json translation file

File: src/main/views/new-account-landing.njk [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        href: "/steps-to-making-your-claim"
+      }) }}
+
+      {{ govukButton({
+        text: "Save for later",
```
Comment (Angela-LW) [inline src/main/views/new-account-landing.njk:34]: same as above, should use the 'saveForLater' string from the common.json translation file

File: src/main/views/new-account-landing.njk (hunk 1 above)
Comment (marcjudge) [reply src/main/views/new-account-landing.njk]: Updated

File: src/main/views/new-account-landing.njk (hunk 2 above)
Comment (marcjudge) [reply src/main/views/new-account-landing.njk]: Updated

**Review by Angela-LW — APPROVED** [review]

**Review by lvo700 — APPROVED** [review]

**Pull request merged**